assets/constants.js

```diff
@@ -1,4 +1,6 @@
-/* eslint-disable */
+/* eslint-disable max-len */
```

Comment:
This one should only be disabled for a specific line.

Reply:
@seanlip updated.